crates/lovely-core/src/patch/regex.rs

```diff
@@ -1,19 +1,24 @@
+use regex_cursor::regex_automata::util::lazy::Lazy;
+use regex_cursor::regex_automata::util::syntax::Config;
+use regex_cursor::regex_automata::Anchored;
 use regex_cursor::Input;
 use regex_cursor::engines::meta::Regex;
+use regex_cursor::regex_automata::util::syntax;
 use regex_cursor::regex_automata::util::interpolate;
 
 use itertools::Itertools;
 use crop::Rope;
 use serde::{Serialize, Deserialize};
 
 use crate::chunk_vec_cursor::IntoCursor;
+
 use super::InsertPosition;
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct RegexPatch {
     pub target: String,
 
-    // The Regex pattern that will be used to both match and create capture groups.
+    // The Regex pattern that will be used to both match and create capture groups.             
```

Comment:
whitespace. not sure how metherul wants to resolve this, do they want to commit an editor config for this?

Reply:
Corrected.